.github/workflows/run-cluster.yaml

```diff
@@ -117,11 +108,13 @@ jobs:
           echo 'Invalid command submitted; command cannot be empty'
           exit 1
         fi
-        ray job submit \
-          --working-dir ${{ inputs.working_dir }} \
-          --address http://localhost:8265 \
-          --runtime-env-json "$ray_env_var" \
-          -- python ${{ inputs.entrypoint_script }} ${{ inputs.entrypoint_args }}
+        rm -rf daft
```

Comment:
Why?

Reply:
https://github.com/Eventual-Inc/Daft/actions/runs/12361613188/job/34499150616

When submitting the repository as the working directory, if `daft` exists inside of the submitted working directory, then python will use that instead of the `uv pip installed`-ed daft wheel. But the `daft` directory doesn't contain the built `.so`, and thus it fails with an error saying some symbol was not able to be resolved.

Reply:
Sorry, edited the link.

Reply:
Updated.